frontend/src/features/public-form/PublicFormProvider.tsx

```diff
@@ -288,6 +304,9 @@ export const PublicFormProvider = ({
         captchaContainerId: containerId,
         expiryInMs,
         isLoading: isLoading || (!!cachedDto?.form.hasCaptcha && !hasLoaded),
+        isMobileSectionSidebarOpen,
+        handleMobileSectionSidebarOpen,
+        handleMobileSectionSidebarClose,
```

Comment:
Same question as above on the explicit listing of the new args, rather than letting `...commonFormValues` pass them implicitly.